docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
```

Comment:
I think it is better to user official image through local Dockerfile, it will be better to allow user override the configuration